terraform/modules/api-proxy-s3-sqs/api-gateway-permissions.tf

```diff
@@ -0,0 +1,104 @@
+resource "aws_iam_role" "social_care_api_cloudwatch" {
+  name               = "${var.resource_name_prefix}-api-gateway-cloudwatch-global"
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "apigateway.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "social_care_api_cloudwatch" {
```

Comment:
You can probably replace this with a managed policy for instance `CloudWatchLogsFullAccess` which will give you the same things but without having to maintain them. You could use a data block to get the policy here instead of defining one.

Reply:
Done in a57fabc